### DISCUSSION TITLE
Where may I ask for a feature?

### DISCUSSION THREAD

**Discussion by frod809** [aristocratos/btop#597] [Ideas]
Hi everyone, hope you are doing ok.

Where may I ask for a feature? I'd really like to see the public ip of my device. 

Thanks.

**Comment by deckstose** [comment]
`curl https://zx2c4.com/ip`

**Reply by deckstose** [reply]
This sounds AI generated.

> Also, I believe that the author of this command would like to incorporate the feature in all three versions, not just in the C++ version

The other version are discontinued in favor of `btop`, so there won't be any updates there I guess.
Also this is not the issue tracker of either `bpytop` or `bashtop`.

> Indeed, the ultimate goal is to incorporate the features across the bash, C++, and Python versions

How do you know that, you are not the author.

>  If you request a feature to this author, chances are higher that the feature will operate on the C++ and Python implementations.

What in the ...?
